ui/components/component-library/text-field/text-field.scss

```diff
@@ -1,10 +0,0 @@
-.mm-text-field {
-  // TOD: remove most of these styles when replaced by ButtonIcon
-  &__button-clear {
-    height: 24px;
-    width: 24px;
-    max-width: 24px;
-    flex: 0 0 24px;
-    margin-left: -24px;
-  }
-}
```

Comment:
Button styles are no longer needed and there are no other styles so we can remove. Still keeping class name as an identifier